concent_api/requirements.lock

```diff
@@ -95,3 +94,4 @@ urllib3==1.23
 vine==1.1.4
 web3==4.2.1
 websockets==5.0.1
+-e ../middleman_protocol
```

Comment:
It's not a part of this pull request but just as a general remark:

It would be more consistent to start using `--editable` instead of `-e`, just like we use the long name for `--find-links`.

Reply:
thanks, fixed